test/spec/provider/element-templates/cmd/ChangeElementTemplateHandler.spec.js

```diff
@@ -2951,6 +3012,7 @@ describe('element-templates - ChangeElementTemplateHandler', function() {
         var serviceTask = elementRegistry.get('ServiceTask_1');
 
         var oldTemplate = {
+          id: 'old',
```

Comment:
Why are these ids needed now?

Reply:
With the change to `applyTemplate` in the helper, we use the internal registry for the lookup of the applied template. If both new and old templates have no ID, the key `undefined` is overridden.

We could monkey-patch the ID in the utility, but I think we should use valid templates for testing instead of doing unrelated magic in the helpers

cf. https://github.com/bpmn-io/bpmn-js-properties-panel/blob/3364078ec56aa663952326b6d755c36949ae51a2/test/spec/provider/element-templates/cmd/ChangeElementTemplateHandler.spec.js#L4486-L4490

Reply:
Nice one, thanks 👍